docs/pages/versions/unversioned/sdk/haptics.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 title: Haptics
-description: A library that provides access to the system's vibration effects on Android and the haptics engine on iOS.
+description: A library that provides access to the system's vibration effects on Android, the haptics engine on iOS, and the Web Vibration API on web platforms.
```

Comment:
```suggestion
description: A library that provides access to the system's vibration effects on Android, the haptics engine on iOS, and the Web Vibration API on web.
```

In context to Android and iOS, web itself is a platform so I think we can omit "platforms" here.